assets/src/js/app.js

```diff
@@ -1,36 +1,53 @@
 import stickyEl from './lib/stickyEl';
-
 const componentName = 'anchor-navigation';
 const baseSelector = '.' + componentName;
 const triggerSelector = baseSelector + '__trigger';
-const navEl = document.querySelector('section' + baseSelector);
-window.AnchorNavigation = {};
-window.AnchorNavigation.settings = null;
-window.AnchorNavigation.init = once(() => {
-  buildNavigation();
-});
-
-if (navEl && !navEl.dataset.attached) {
-  window.AnchorNavigation.settings = drupalSettings.anchorNavigation;
-  window.AnchorNavigation.init();
-  navEl.dataset.attached = true;
+let navEls = [];
```

Comment:
```suggestion
const navEls = [];
```
This should stay a constant